src/toil/jobStores/abstractJobStore.py

```diff
@@ -376,6 +381,8 @@ def _defaultExportFile(self, otherCls, jobStoreFileID, url):
         """
         with self.readFileStream(jobStoreFileID) as readable:
             otherCls._writeToUrl(readable, url)
+        if getattr(jobStoreFileID, 'executable', False):
```

Comment:
@adamnovak I was also thinking of doing this inside of `writeToUrl` instead by passing in the executable bit as an arg. Do you think that would be a better approach compared to what I currently have? I'm just wondering since we talked about keeping the implementation inside `writeToUrl` and `readToUrl`.

Reply:
Yeah, you probably have to keep this inside the `_writeToUrl` that actually writes local files.

Here, you don't know that the URL even is a `file://` URL. So if someone tries to export to `s3://bucket/some/place/in/a/bucket`, you are going to grab the `url.path` (`some/place/in/a/bucket`) and try to chmod that executable as if it were a local filename. That will probably throw an error, if you're lucky. If you're unlucky, that path will happen to exist and you'll be setting the executable bit on whatever that file happens to be on the local machine.